crates/sp-domains-fraud-proof/src/lib.rs

```diff
@@ -190,3 +193,12 @@ impl FraudProofVerificationInfoResponse {
         }
     }
 }
+
+sp_api::decl_runtime_apis! {
+    pub trait FraudProofsApi<DomainHeader: HeaderT> {
+        /// Submits the fraud proof via an unsigned extrinsic.
+        fn submit_fraud_proof_unsigned(
+            fraud_proof: FraudProof<NumberFor<Block>, Block::Hash, DomainHeader>,
+        );
+    }
+}
```

Comment:
This is duplicated with #2169, would appreciate to get #2169 merged first, thanks!